tests/php/src/ReaderThemeLoaderTest.php

```diff
@@ -214,6 +215,7 @@ public function test_override_theme() {
 		if ( ! wp_get_theme( $active_theme_slug )->exists() || ! wp_get_theme( $reader_theme_slug )->exists() ) {
 			$this->markTestSkipped();
 		}
+		wp_reset_query(); // phpcs:ignore WordPress.WP.DiscouragedFunctions.wp_reset_query_wp_reset_query
```

Comment:
Why was this needed?

Reply:
I guess same reason that this is needed above: `$GLOBALS['wp_the_query'] = $GLOBALS['wp_query']`